htdocs/core/lib/functions.lib.php

```diff
@@ -8905,6 +8906,7 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 			'__MYCOMPANY_ADDRESS__' => $mysoc->address,
 			'__MYCOMPANY_ZIP__'     => $mysoc->zip,
 			'__MYCOMPANY_TOWN__'    => $mysoc->town,
+			'__MYCOMPANY_STATE__'    => getState($mysoc->state_id),
```

Comment:
Simple substition key `__XXX__` must never call methods that do SQL request.
If we need this we must use lazy load substitution key with syntax 
`__XXX__@lazyload` 
See https://wiki.dolibarr.org/index.php?title=Variable_substitution_system#How_it_work

But in your case, may be you can use directly
`'__MYCOMPANY_STATE__' => $mysoc->state,`

Reply:
Perfect, I've already applied the change. This is easier than the initial proposal. Thank you!